### DISCUSSION TITLE
Signing in with Recovery code doesn’t log into the app

### DISCUSSION THREAD

**Discussion by tekrondo** [community/community#44961] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.96.0 (88967961)
Device: iPhone15,3 (iOS 16.2)

When I sign into the mobile app using the recovery key, it logs in using the browser in the app,  it never returns to the native app.
</details>

**Comment by eliperkins** [comment]
Hey @tekrondo , thanks for this feedback! Can I ask a couple of questions to gain a better understanding here?

- You mention `When I sign into the mobile app using the recovery key`. For my understanding, can you clarify where you're tapping within the login flow to use recovery keys to sign in?
    - For context, [recovery keys are meant for account recovery when you've lost all other 2FA methods](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials), so I wouldn't anticipate them to complete the OAuth flow to log you into GitHub Mobile itself. I'd love to learn  more here!
- You mention `it logs in using the browser in the app, it never returns to the native app`. Does this mean that you're using GitHub Mobile to recover your account? Or you're using recovery codes from a `github-recovery-codes.txt` to log into GitHub Mobile?

Thanks for your help in diagnosing this one!
